static-site/app/styles/browserCompatability.css

```diff
@@ -0,0 +1,446 @@
+/* HTML5 display definitions
```

Comment:
How hard would it be to structure the commits such that we can track these "new" files?

This stylesheet is a duplicate (with some commented out sections removed¹). Can we not simply make this a file move and have the `/pages/*` files use it from its new location? bootstrap.css is a straight copy and globals.css looks like a conversion from `theme.js` (and maybe others), so I can see here the desire to maintain things in parallel until the entire `/pages` section is removed.

This applies to other files in this PR as well, e.g. `static-site/components/people/teamMembers.ts` is largely a copy of `static-site/src/components/People/teamMembers.js`

¹ `git diff master:./static-site/src/styles/browserCompatability.css HEAD:static-site/app/styles/browserCompatability.css `

Reply:
> How hard would it be to structure the commits such that we can track these "new" files?

I think it would be trickier than it's worth, because at the end of all this, the original/source files will be removed, along with all the Pages Router bits. 

Duplicating them now, at the beginning, and scoping them only to the new App Router section, means that I don't have to worry about some change having a temporary bad side-effect on Pages Router pages, the way I would need to if I changed those pages to reference the new copies.

Reply:
If the conversion for the entire site will be done in one PR, squashing all commits would be an option. We haven't done this much in the team, but we also haven't had a migration of this scope before!

Reply:
I don't think squashing everything into one monster commit is going to improve the understandability here; I do think keeping all changes related to one "step" of the process is the best balance between the not-always-aligned goals of "make sure things work at each step", "make it possible to see the changes", and "don't waste massive amounts of time jumping through git hoops".

Reply:
Re: https://github.com/nextstrain/nextstrain.org/pull/1032#issuecomment-2463428609

> Not sure why I can't `quote reply` this one comment, but...

Oh I replied to an existing thread so you have to go back to the thread to add a comment. GitHub UI doesn't do a great job of showing this... copying back to the thread.

> > My biggest concern is the lack of clarity around where the duplicates are.
> 
> I guess I don't see this as a huge issue, because when all this work is done, all the duplicates will be gone, as all the "old" components will be removed/replaced — more specifically, `static-site/pages` and `static-site/src` will no longer be present.
> 
> If the work is introducing multiple duplicates where there is currently only a single copy, I would consider that a bug/mis-implementation — but I don't think that's the case.
> 
> The only way to do this without having some transient duplication would be to convert the entire `static-site` directory at once, and even then, you'd have to decide between the equally unpalatable options of "have some commits where the site is broken" and "have some commits that are far too large and change multiple things at once due to shared component usage". Compared to those two options, I think having some duplication present in the tree in some commits is the least bad option.

Sorry if I wasn't clear – I'm not suggesting to remove the duplication. It's obviously unavoidable with the iterative approach! I'm suggesting to improve the clarity. I don't see anything obvious that maps copy A to copy B unless the filenames are similar, which is not always the case.